src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,49 @@
+import cn from 'classnames';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[] | [];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <>
+      {todos && (
+        todos.map(todo => (
+          <div
+            data-cy="Todo"
+            className={cn('todo', { active: todo.completed })}
+          >
```

Comment:
Don't forget to add a key attribute.
Use key attribute correctly (read [here](https://medium.com/blackrock-engineering/5-common-mistakes-with-keys-in-react-b86e82020052) for more details)